docs/examples/tukey-outlier-detection.py

```diff
@@ -0,0 +1,47 @@
+"""
+Tukey Outlier Detection
+=======================
+
+Identifying outliers in time series using
+Tukey outlier detection.
+"""
+
+# %%
+# Identifying and removing outliers from PV sensor time series
+# data allows for more accurate data analysis.
+# In this example, we demonstrate how to use
+# :py:func:`pvanalytics.quality.outliers.tukey` to identify and filter
+# out outliers in a time series.
+# We use a normalized time series example provided by the PV Fleets Initiative.
+# This example is adapted from the DuraMAT DataHub
+# clipping data set:
+# https://datahub.duramat.org/dataset/inverter-clipping-ml-training-set-real-data
+
+import pvanalytics
+from pvanalytics.quality.outliers import tukey
+import matplotlib.pyplot as plt
+import pandas as pd
+import pathlib
+
+# %%
+# First, we read in the ac_power_inv_7539_outliers example. Normalized AC power
+# is represented by the "value_normalized" column. There is a boolean column
+# "outlier" where inserted outliers are labeled as True, and all other values
+# are labeled as False.
```

Comment:
I understand what is being done but wonder if we're being a bit terse here.  The data contain normalized AC power (or energy?) at 15 minute intervals, where power is normalized by the inverter's AC capacity (?). For the purpose of illustrating outlier detection, we have overwritten a few lines of data with extreme values.  Is that correct?

Reply:
@cwhanse that is correct. The data is actually min-max normalized and is in 15-minute intervals. I added a smattering of extreme outliers to illustrate the function performance when detecting outliers. I can make this more explicit in our documentation if that would help

Reply:
@cwhanse I made some updates in the commenting per your recommendations, and made the outlier insertion commenting more explicit. Let me know if this works and I'll merge!